src/services/api/peers.js

```diff
@@ -93,11 +94,24 @@ app.factory('Peers', ($timeout, $cookies, $location, $q, $rootScope) => {
      * @private
      * @memberOf Peer
      * @method check
+     * @notifyUser {bool} - Should an error toast be displayed on error?
      */
-    check() {
+    check(notifyUser) {
```

Comment:
The thing is that this should be only on login. After login when this fails, the whole ui get's grey and loading spinner is displayed, so the user is notified in either case. I changed the param name accordingly.